docs/src/components/Markdown.css

```diff
@@ -18,8 +18,35 @@ html[dir="rtl"] .Markdown ul + pre {
 }
 
 .Markdown a {
-  color: rgb(0, 0, 0);
+  color: var(--g-navy);
   font-weight: bold;
+}
+
+.Markdown a:hover {
+  color: var(--g-midnight);
   text-decoration: none;
 }
 
+.Markdown h2 {
+  align-items: baseline;
+  display: flex;
+  scroll-margin-top: 75px;
+}
+
+.Markdown h3 {
+  color: var(--g-colorGray200);
+}
+
+.anchor {
```

Comment:
@ayeshakmaz could we also scope these to `.Markdown`?

`.Markdown .anchor`
and
`.Markdown .headerLink`

Reply:
Sure thing, any reason we don't use Sass?

Reply:
@ayeshakmaz we haven't need it so far. Ideally these would be Gestalt components instead of custom HTML tags.